openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -131,7 +131,17 @@
                     options: options.language_options,
                     helpMessage: '',
                     persistChanges: true
+                }),
+
+                new FieldsView.ReadonlyFieldView({
```

Comment:
As noted in HipChat, it would be great if @andy-armstrong could provide direction on the following before you are gone.

Does it make sense to introduce a date field that knows how to fill out the data attributes appropriately to deal with localization?
https://github.com/edx/edx-platform/blob/5ed622d6d53aa90ead230139b3a9d6d9c97b093e/lms/static/js/views/fields.js#L283-L283